transit_odp/timetables/transformers.py

```diff
@@ -161,9 +160,12 @@ def transform(self) -> TransformedData:
             ) = transform_service_pattern_to_service_links(  # noqa: E501
                 service_patterns, route_to_route_links, route_links
             )
-            service_patterns = service_patterns.drop_duplicates(
+            service_patterns = service_patterns.reset_index().drop_duplicates(
```

Comment:
Multiple entries with different service pattern id for the same vehicle journey seems to be stored in db. Also most of these journeys are missing

![image](https://github.com/department-for-transport-BODS/bods/assets/95031756/f21d1e39-06c1-4b73-b7bd-65c63123ef42)